manager/testdata/e2e/module-implicit-copy-s3-to-s3.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: app.m4d.ibm.com/v1alpha1
+kind: M4DModule
+metadata:
+  name: implicit-copy-s3-to-s3-batch # module name
+  namespace: m4d-system  # In future, will be a parameter defined by admin
+  labels:
+    name: implicit-copy-s3-to-s3-batch # module name
+    version: 1.0.0 
+  annotations:
+    author: "Example Author (author@example.com)"
+    summary: "a short description of this module"
+    description: "a longer description of this module that might e.g. be used in contracts"
+spec:
+  type: service # one of: job, service, configuration
```

Comment:
Job. 

We may want to omit this field from the spec, we don't seem to need it.